problem_builder/mentoring.py

```diff
@@ -1245,3 +1245,24 @@ def author_edit_view(self, context):
         fragment.add_javascript_url(self.runtime.local_resource_url(self, 'public/js/container_edit.js'))
         fragment.initialize_js('ProblemBuilderContainerEdit')
         return fragment
+
+    def student_view_data(self, context=None):
+        children_contents = []
+
+        for child_id in self.children:
+            child = self.runtime.get_block(child_id)
+
+            if child:
+                child_contents = child.student_view_data(context)
+                children_contents.append(child_contents)
+        return {
+            'self': {
+                'active_step_safe': self.active_step_safe,
```

Comment:
Rename `active_step_safe` to `active_step`. The fact that it's using `active_step_safe` method to obtain the active step is just an internal implementation detail.